parm/archive/enkf_restarta_grp.yaml.j2

```diff
@@ -65,7 +65,7 @@ enkf_restarta_grp:
         {% set rst_delta = "+6H" | to_timedelta %}
         {% endif %} # DOIAU_ENKF
         {% set rst_time = current_cycle | add_to_datetime(rst_delta) %}
-        - "{{ COMIN_OCEAN_RESTART_MEM | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ rst_time | strftime("%H") }}0000.MOM.res.nc"
+        - "{{ COMIN_OCEAN_RESTART_MEM | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ rst_time | strftime("%H") }}0000.MOM.r*.nc"
```

Comment:
What file(s) is this actually capturing?  The only files I see in recent `C96C48mx500_S2SW` tests end with `MOM.res.nc`.  Is this changing?

Reply:
I note that you are running a `gfsv17/C1152mx025_S2SW_rdhpcs` test case in the description.  May I take a look at that case?